pkg/remoteconfig/updater.go

```diff
@@ -155,7 +162,7 @@ func (r *RemoteConfigUpdater) Start(apiKey string, site string, clusterName stri
 	rcClient, err := client.NewClient(
 		rcService,
 		client.WithAgent("datadog-operator", version.Version),
-		client.WithProducts(state.ProductAgentConfig),
+		client.WithProducts(state.ProductAgentConfig, crdRcProduct),
```

Comment:
Could it be a single product ?
In both case, this is to manipulate agent config

Reply:
These are completely different products in remote config configuring different parts of the agent. This here is referring to the RC product

Reply:
What I'm afraid of is that we have 2 RC products to configure the agent. What happens if you enable CSM VM through AgentConfig RC product but set `features.sbom.containerImage.enabled` to true in the CRD RC product ?

Reply:
I'm also confused by this as well. `ProductOrchestratorK8sCRDs` doesn't even imply whole Orchestrator Explorer feature, so are we going to have `ProductOrchestrator` later? 
With this approach we may end up with 30+ products. What are the implication of such granularity? 
cc @timothyalexandersoftware

Reply:
I think I'd need to understand a bit more about the existing product(s) using `ProductAgentConfig` and how they are configuring the agent, as compared to what Josh is introducing. If these are truly just different configurations for the agent that are getting configured via two different products, there is indeed an opportunity to consolidate. 

This would add a bit of complexity for managing the release of different features though -- e.g. if both features use the same product, it would be non-trivial to have a killswitch to turn off the K8s CRDs product/feature without affecting the security product and vice versa.

Reply:
Checked with @timothyalexandersoftware and for now this approach is acceptable but we may have to revisit it later and consolidate these products into one.